docs/integrations/microsoft-azure/azure-app-service-plan.md

```diff
@@ -14,7 +14,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 For Azure App Service Plan, you can collect the following metrics:
 
-* **Platform Metrics for Azure App Service Plan**. These metrics are available in [Microsoft.Web/serverfarms](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-web-serverfarms-metrics) namespace. For more information on supported metrics, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/app-service/web-sites-monitor#understand-metrics). App Service plan metrics are available only for plans in Basic, Standard, and Premium tiers.
+- **Activity logs**. Provide insight into any subscription-level or management group level events that have occurred in the Azure. To learn more, refer to [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema).
+
+- **Platform Metrics for Azure App Service Plan**. These metrics are available in the [Microsoft.Web/serverfarms](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-web-serverfarms-metrics) namespace. For more information on supported metrics, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/app-service/web-sites-monitor#understand-metrics). App Service plan metrics are available only for plans in Basic, Standard, and Premium tiers.
```

Comment:
lets add the screenshot for diagnostic settings if not present

Reply:
Done

Reply:
screenshot file is it already pushed ?